src/FOHClasses/Waiter.java

```diff
@@ -0,0 +1,124 @@
+package FOHClasses;
+
+public class Waiter {
+    private final int staffID;
+    private String name;
+    private int pin;
+    private String phoneNumber;
```

Comment:
Not sure if it's necessary to store this information in front of house. I believe management will store it as part of their employee management.